packages/@okta/vuepress-theme-prose/assets/css/components/_horizontalDivider.scss

```diff
@@ -0,0 +1,6 @@
+.horizontal-divider{
+  height: 1px;
+  width: 100%;
+  background-color: #E0E0E0;
```

Comment:
seems this color can be taken from `_variables-theme.scss`

Reply:
there is a similar